client/src/components/organisms/RequestForm.tsx

```diff
@@ -223,9 +223,6 @@ const RequestGroupForm: FunctionComponent<Props> = (props: Props) => {
   const onRequestGroupInputChange = (newRequestGroupInput: string) => {
     setChangeMade(true);
     setRequestGroupInputGroup(newRequestGroupInput)
-    if (!newRequestGroupInput) {
```

Comment:
Deleted because it violated expected behaviour.

Here's the expected behaviour:
+ Previously: user selected a RequestGroup `A` in the SearchableDropdown and the drop down is closed
+ Actions:
  1. User focuses on SearchableDropdown
  2. User puts empty search string `""` in SearchableDropdown
  3. User unfocuses from Searchable without selecting any new request group
  4. Selected request group displayed by SearchableDropdown should still be `A`